lib/dispatch/settings/json_static_file_client.ex

```diff
@@ -62,7 +62,7 @@ defmodule Dispatch.Settings.JSONStaticFileClient do
     %State{
       learners: Map.get(configuration, "learners", %{}),
       experts: Map.get(configuration, "experts", %{}),
-      blacklist: Map.get(configuration, "blacklist", [])
+      blocklist: Map.get(configuration, "blocklist", [])
```

Comment:
Have you considered adding a fallback to `blacklist` if `blocklist` is not present to avoid the breaking change?

Reply:
I did but with the `Agent` and the code in place it was a lot more work than expected. I don’t know if it is worth the effort, I’ll give it another try before merging this PR.

Reply:
It was pretty easy finally :)